docs/user_guide/Upgrade_SIMP/Version_Maps/6.5.0_6.6.0.inc

```diff
@@ -0,0 +1,228 @@
+
+Set simp::sssd::client::ldap_server_type
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If :term:`LDAP:` and :term:`SSSD` are both used, you must set the type of LDAP
+server in :term:`hiera` using the :code:`simp::sssd::client::ldap_server_type`
+parameter.
+
+If the system is using an OpenLDAP server, you would set this option to
+:code:`plain`. For 389-DS-derived systems (389-DS, FreeIPA, RHDS, etc...) you
+would set this to :code:`389ds`. An example is provided below
+
+.. code-block:: yaml
+
+   # The upstream server is OpenLDAP
+   simp::sssd::client:ldap_server_type: 'plain'
+
+   # The upstream server is 389-DS
+   simp::sssd::client:ldap_server_type: '389ds'
+
+Requirement for SSSD >= 1.16.0
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In systems using :program:`sssd` prior to 1.16.0, the daemon would fail to start
+without a :code:`LOCAL` domain. For security and supportability reasons, SIMP no
+longer supports :program:`sssd` prior to 1.16.0. Likewise, the
+:code:`simp::sssd::client` class no longer configures a :code:`LOCAL` domain.
+
+If you have configured a :code:`LOCAL` domain for some other reason you must:
+
+* Remove :code:`LOCAL` from the list of domains in :term:`hiera` from
+  :code:`sssd::domains:`
+* Disable the creation of the local domain by :code:`simp::sssd::client`
+
+.. code-block:: yaml
+
+   # These are examples of the sssd::domains entry you might see in hiera.
+
+   # LOCAL is the only entry, remove everything
+   sssd::domains:
+     - LOCAL
+
+   # LOCAL is not the only entry, only remove LOCAL
+   sssd::domains:
+     - LOCAL #remove this
+     - MY_DOMAIN
+     - SOMETHING_ELSE
+
+If there are errors in systems before SIMP 6.6.0 has been fully installed, the
+creation of the :code:`LOCAL` domain can be prevented by setting the following
+in :term:`hiera`:
+
+.. code-block:: yaml
+
+   simp::sssd::client::local_domain: false
+
+NTP Changes
+"""""""""""
+
+The :code:`simp_options::ntpd::servers` parameter has been changed to
+:code:`simp_options::ntp::servers`.
+
+Both :program:`ntpd` and :program:`chronyd` use
+:code:`simp_options::ntp::servers` as the default.
+
+In :term:`hiera`, users must:
+
+* Rename :code:`simp_options::ntpd::servers` to :code:`simp_options::ntp::servers`
+* Create an :code:`alias` for backwards compatibility
+
+Old :term:`hiera`:
+
+.. code-block:: yaml
+
+   simp_options::ntpd::servers:
+   - server1.time.server.org
+   - server2.time.server.org
+   - server3.time.server.org
+   - 127.127.1.0
+
+New :term:`hiera`:
+
+.. code-block:: yaml
+
+   simp_options::ntp::servers:
+   - server1.time.server.org
+   - server2.time.server.org
+   - server3.time.server.org
+   - 127.127.1.0
+   # The following is work around needed during upgrade
+   simp_options::ntpd::servers: "%{alias('simp_options::ntp::servers')}"
+
+
+Update Yum Repositories
+"""""""""""""""""""""""
+
+Migrate to the New Format
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**The following replaces Step 1**, `Update the YUM Repositories`, of the
```

Comment:
I would make 'The following replaces Step1' all uppercase, as it is really important.

Reply:
It's already in **bolt** but some extra push wouldn't hurt.